### DISCUSSION TITLE
Config file (at least InfluxDB2 section) is not read correctly. All fields are read as `None`

### DISCUSSION THREAD

**Discussion by mvicomoya** [nicolargo/glances#2654] [General]
**Describe the bug**
All fields from InfluxDB2 config section are read as `None`, even though they have valid values.

This is happening with `latest` docker image from https://hub.docker.com/r/nicolargo, but not with `3.4.0.3`

**To Reproduce**
1. Configure InfluxDB2 export for glances
2. Start Glances using the latest `dev` docker image, using the following options:
```
GLANCES_OPT: "-q --export influxdb2 --time 10"
```

**Expected behavior**
I should see glances data flowing to influxdb. I should NOT see this log coming from glances:
```
2024-01-08 23:58:47,487 -- ERROR -- The batch item wasn't processed successfully because: Missing the required parameter `org` when calling `post_write`
```

**Screenshots**
N/A

**Environement**
Docker image from https://hub.docker.com/layers/nicolargo/glances/latest

The InfluxDB2 section of my config file looks like:
```
[influxdb2]
host=subdomain.domain.com
port=443
protocol=https
org=<my org ID>
bucket=<my bucket ID>
token=<write token>
```

**Additional context**
Adding `--debug` to Glances options, I see this in the logs:
```
2024-01-09 00:03:22,402 -- DEBUG -- influxdb2 parameters: {'host': None, 'port': None, 'user': None, 'password': None, 'org': None, 'bucket': None, 'token': None, 'protocol': None, 'prefix': None, 'tags': None, 'interval': None}
```

Same configuration file without changes works correctly with the `3.4.0.3`

**Comment by mvicomoya** [comment]
Might be related to #2649

**Comment by nicolargo** [comment]
Your glances.conf file is not readed correctly (or not found). Read the documentation here: https://github.com/nicolargo/glances#docker-the-fun-way

**Comment by nicolargo** [comment]
Try to pull the latest Docker image (pushed today). I patch it to force a default glances.conf file.

**Reply by mvicomoya** [reply]
@nicolargo, thanks for the suggestion. Changing the config mount point and explicitly passing the config path with the `-C` option did indeed do the trick.

Is there any communication channel I could subscribe to in order to get notified of future backwards incompatible changes like this?

**Comment by nicolargo** [comment]
Hi !

All breaking change should be listes in the News file (uodated for each
release).
